### PR TITLE
Expose aruco detection parameters and tweak some values

### PR DESCRIPTION
**Pull request by jim-v**
Another step towards addressing #2

### CODE REVIEW

**Review by rohbotics — APPROVED** [review]
Looks good, we should make these dynamic reconfigurable to make tuning easier.

**Comment by jim2v** [comment]
Yes, good idea, @rohbotics .  Can you take a look, please.

File: aruco_detect/src/aruco_detect.cpp [hunk 1]
```diff
@@ -233,27 +261,6 @@ FiducialsNode::FiducialsNode(ros::NodeHandle & nh) : it(nh)
...
-    nh.param<double>("maxErroneousBitsInBorderRate", detectorParams->maxErroneousBitsInBorderRate, 0.04);
-    nh.param<int>("minDistanceToBorder", detectorParams->minDistanceToBorder, 3);
-    nh.param<double>("minMarkerDistanceRate", detectorParams->minMarkerDistanceRate, 0.05);
-    nh.param<double>("minMarkerPerimeterRate", detectorParams->minMarkerPerimeterRate, 0.1); /* default 0.3 */
-    nh.param<double>("maxMarkerPerimeterRate", detectorParams->maxMarkerPerimeterRate, 4.0);
-    nh.param<double>("minOtsuStdDev", detectorParams->minOtsuStdDev, 5.0);
-    nh.param<double>("perspectiveRemoveIgnoredMarginPerCell", detectorParams->perspectiveRemoveIgnoredMarginPerCell, 0.13);
-    nh.param<int>("perspectiveRemovePixelPerCell", detectorParams->perspectiveRemovePixelPerCell, 8);
-    nh.param<double>("polygonalApproxAccuracyRate", detectorParams->polygonalApproxAccuracyRate, 0.01); /* default 0.05 */
-
```
Comment (rohbotics) [inline aruco_detect/src/aruco_detect.cpp]: Keep the parameter versions, otherwise we can't load these settings from a yaml file.

**Review by rohbotics — CHANGES_REQUESTED** [review]

**Review by rohbotics — APPROVED** [review]
Looks good, thanks!

**Pull request merged**